publisher-utils/monitor/price/monitor-price.py

```diff
@@ -0,0 +1,91 @@
+import asyncio
+import datetime
+import os
+import time
+import traceback
+
+import requests
+from pontis.core.client import PontisClient
+from pontis.core.const import DEFAULT_AGGREGATION_MODE
+from pontis.core.utils import currency_pair_to_key, str_to_felt
+from pontis.publisher.fetch import fetch_coingecko
+
+# Behavior: Ping betteruptime iff all is good
+
+
+PRICE_TOLERANCE = 0.1  # in percent
+TIME_TOLERANCE = 600  # in seconds
+
+
+async def main():
+    assets = [
```

Comment:
This is duplicated in sample-publisher/all, we will probably want to move this to a config. Ultimately probably want a fall-back in pontis-package?